designs/images/cluster_reconcile.plantuml

```diff
@@ -0,0 +1,61 @@
+@startuml
```

Comment:
This is dope 👀

Reply:
A rendering would be nice

Reply:
Oh, now I see the png and that shows up when I view the above doc, cool

Reply:
cluster_lifecycle.plantuml?

Reply:
@TerryHowe 
There are two images
* `cluster-lifecycle.png` is the high level one (the first in the doc). This one wasn't generated with `plantuml` so we don't have a "source file"
* `cluster_reconcile.png` is the cluster reconcile flow diagram (second image in the doc). This file is the source for that one